.github/workflows/openapi-diff.yml

```diff
@@ -0,0 +1,62 @@
+name: CI
+on: [pull_request]
+jobs:
+  openapi-diff:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout HEAD
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.head_ref }}
+          path: head
+      - name: Checkout BASE
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ github.base_ref }}
+          path: base
+      - name: Run docker manually
+        id: oasdif_changelog
+        run: | # Capture changelog as a multiline output
+          echo "changelog<<EOF" > $GITHUB_OUTPUT
+          docker run --rm \
+            --workdir ${{ github.workspace }} \
+            --volume ${{ github.workspace }}:${{ github.workspace }}:rw \
+            -e GITHUB_WORKSPACE=${{ github.workspace }} \
+            tufin/oasdiff changelog --composed \
+            'base/openapi/specs/*.yaml' \
+            'head/openapi/specs/*.yaml' \
+            >> $GITHUB_OUTPUT
+          echo "EOF" >> $GITHUB_OUTPUT
+      - name: Get summary
+        id: oasdif_summary
+        run: |
+          echo summary=$(echo "${{ steps.oasdif_changelog.outputs.changelog }}" | head -1 ) \
+          >> $GITHUB_OUTPUT
+      - name: Post changes as comment
+        uses: thollander/actions-comment-pull-request@v2
+        with:
+          message: |
+            ## OpenAPI Changes
+
+            <details>
+            <summary>Show/hide ${{ steps.oasdif_summary.outputs.summary || 'No detectable change.' }}</summary>
```

Comment:
Some changes aren't reported by tufin/oasdiff. For example, changing a response body property from `nullable: true` to `nullable: false` seems not to be detected. (I'm not sure if that's intentional on their part.)

Reply:
Changing a response body property from `nullable: true` to `nullable: false` should be safe because `int`  matches the type `int | null`. It's on the list of safe changes in their example doc as well: https://github.com/Tufin/oasdiff/blob/main/BREAKING-CHANGES-EXAMPLES.md

Reply:
It didn't surprise me that it's not a breaking change. It surprises me it's not reported at all, e.g., undo the "info" category. But... Not a big deal.